swaglyrics/__main__.py

```diff
@@ -29,19 +31,40 @@ def main():
 
 	parser.add_argument('-t', '--tab', action='store_true', help='Display lyrics in a browser tab.')
 	parser.add_argument('-c', '--cli', action='store_true', help='Display lyrics in the command-line.')
+	parser.add_argument('-cr', '--chrome', action='store_true', help='Display lyrics from browser.')
 
 	args = parser.parse_args()
+	app.template_folder = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'templates')
```

Comment:
Why do you need to do this if the tab option has not been called?

Reply:
Imported it along with static folder. kept it there for code aesthetics

Reply:
You shouldn't be needing it here.